test_conformance/relationals/test_comparisons_fp.cpp

```diff
@@ -385,6 +402,20 @@ int RelationalsFPTest::test_equiv_kernel(unsigned int vecSize,
                         return -1;
                     }
                 }
+                else if (std::is_same<T, cl_half>::value)
+                {
+                    bool in_denorm = IsHalfSubnormal(inDataA[i * vecSize])
+                        || IsHalfSubnormal(inDataB[i * vecSize]);
```

Comment:
Same here:

bool in_denorm = IsHalfSubnormal(inDataA[i * vecSize + j])
                        || IsHalfSubnormal(inDataB[i * vecSize + j]);

Reply:
This works now